Update HASS temp_stat_tpl min-max enforcement

Sprinterfreak opened the pull request:
Allow values including min and max values instead of just greater and smaller then. Replaces ifelse by Jinja embedded min and max functions.

Fixes HASS ERROR "Could not parse temperature from" if set temp equals min or max temp.

Sprinterfreak commented:
probably we could get rid of the surrounding ifelse by utilizing
`|default(" + (String)convertCelsiusToLocalUnit(22, useFahrenheit) + ")`

mayankst commented:
@Sprinterfreak how come this and your other PR has same change? 
![image](https://user-images.githubusercontent.com/67297265/204621148-0e3cfca8-7ab2-42ae-aec8-61468afd194a.png)

Sprinterfreak commented:
Convinience for the maintainer. Updated refs/master and rebased on-top of current changes to keep the log clean

Sprinterfreak commented:
Accidentally deleted the wrong changes while preparing the second PR :(

It was difficult to work from a clean master, because to be able to test it, I needed the first PR applied for the software to actually work.

Sprinterfreak committed:
Update HASS temp_stat_tpl min-max enforcement

Allow values including min and max values instead of just greater and
smaller then. Replaces ifelse by Jinja embedded min and max functions.

Fixes HASS ERROR "Could not parse temperature from"
if set temp equals min or max temp.

Sprinterfreak commented:
Not so easy to prepare multiple clean PR to one file if there are multiple independent things broken which have to be fixed first in order to be able to test.
